pkgs/development/compilers/rust/rustc.nix

```diff
@@ -105,6 +105,11 @@ stdenv.mkDerivation {
     # On Hydra: `TcpListener::bind(&addr)`: Address already in use (os error 98)'
     sed '/^ *fn fast_rebind()/i#[ignore]' -i src/libstd/net/tcp.rs
 
+    # https://github.com/rust-lang/rust/issues/39522
+    echo removing gdb-version-sensitive tests...
+    find src/test/debuginfo -type f -execdir grep -q ignore-gdb-version '{}' \; -print -delete
+    rm src/test/debuginfo/{borrowed-c-style-enum.rs,c-style-enum-in-composite.rs,gdb-pretty-struct-and-enums-pre-gdb-7-7.rs,generic-enum-with-different-disr-sizes.rs}
```

Comment:
Agreed, it is not super critical for us and I don't feel like we would be in the position to fix this anyway.

Reply:
Aren't the tests still disabled? https://github.com/NixOS/nixpkgs/commit/323fe3a305f64bb23e6ea75105654d307a1fa9e9

Reply:
Indeed did you enable them again locally?

Reply:
@symphorien Do you still have the error, or was it something useless?

Reply:
This addition is useless since tests are disabled, but I thought it may be useful to leave it here if someone tries to reenable tests in the future. I can remove this if you wish so.